runtime/devnet/src/lib.rs

```diff
@@ -252,75 +254,15 @@ impl Contains<RuntimeCall> for FilteredCalls {
 
 /// A type to identify allowed calls to the Runtime from contracts. Used by Pop API
 pub struct AllowedApiCalls;
```

Comment:
Could be moved to suggested `/configs/api.rs` module. Or we just have an `api.rs` module for this, the state read enum and the pallet configs so it is treated as one api from the runtime maintenance perspective.

Reply:
see refactor: devnet runtime